libvmi/os/windows/peparse.c

```diff
@@ -403,13 +403,15 @@ peparse_get_idd_rva(
 
         if(*optional_header_type == IMAGE_PE32_MAGIC) {
             struct optional_header_pe32 *oh_pe32_t = (struct optional_header_pe32 *)optional_header;
-            rva = oh_pe32_t->idd[entry_id].virtual_address;
+            if(oh_pe32_t->number_of_rva_and_sizes >= entry_id)
```

Comment:
Braces around the single statements here would make the intent to not include the `goto` clear, since the other tests of this form include the `goto`.